src/main/java/LottoApplication.java

```diff
@@ -0,0 +1,9 @@
+public class LottoApplication {
+
+	public static void main(String[] args) {
+		LottoView view = new LottoView();
+		LottoController controller = new LottoController(view);
```

Comment:
사소하지만 지금은 view 객체를 굳이 변수로 할당할 필요는 없는 것 같은데, 생성과 동시에 생성자 파라미터로 주입하는 것이 더 깔끔해보이네요. 😃
```suggestion
		LottoController controller = new LottoController(new LottoView());
```

Reply:
view 의 메소드들을 모두 static 메소드로 변경하여 Controller 단에서 View 인스턴스를 받지 않도록 하였습니다.
https://github.com/next-step/java-lotto-kakao/pull/14/commits/6f04aaba2389f594f78dda92e9b517c84bd37d6b

Reply:
네 LottoView가 인스턴스 변수를 가지지 않기 때문에 굳이 객체로 생성할 필요가 없는 것이 맞습니다. 🙂👍
좋은 방향인 것 같아요. 😃

조금 더 나아가서 `LottoView`가 객체를 생성할 수 없도록 기본 생성자의 접근 제한자를 `private`로 변경해줘도 좋을 것 같아요. 😉